Userland/Services/WebWorker/PageHost.h

```diff
@@ -32,6 +32,7 @@ class PageHost final : public Web::PageClient {
     virtual Web::CSS::PreferredContrast preferred_contrast() const override;
     virtual Web::CSS::PreferredMotion preferred_motion() const override;
     virtual void paint_next_frame() override {};
+    virtual void process_screenshot_requests() override {};
```

Comment:
This should be `virtual void process_screenshot_requests() override { }` Weird that the clang-format hook didn't catch this?